docs/tutorial/index.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import Helmet from "react-helmet"
+import Link from "gatsby-link"
+
+import Container from "../components/container"
+
+export default () => (
+  <Container>
+    <Helmet>
+      <title>Tutorial</title>
+    </Helmet>
+    <h1 css={{ marginTop: 0 }}>Gatsby.js Tutorial</h1>
+   <p>
+      We’re so happy you decided to try using Gatsby. This tutorial has seven
+      parts that'll walk you from getting started developing and building Gatsby
+      sites to deploying a finished and polished high performance static PWA.
+    </p>
+    <p>
+      This tutorial is for <em>everyone</em>! You do not need to be a programmer
+      or React.js expert. We'll walk you through things.
+    </p>
+
+    <ol>
+      <li>
+        <Link to="/tutorial/part-zero/">Programming basics (Part Zero)</Link>. 
+        If you are new to programming, here's an introduction to some basics like terminal, 
```

Comment:
I think a more active construction would be better e.g. "go here for introductions to..."